src/utilities/buildASTSchema.js

```diff
@@ -132,10 +132,9 @@ function getNamedTypeNode(typeNode: TypeNode): NamedTypeNode {
  * If no schema definition is provided, then it will look for types named Query
  * and Mutation.
  *
- * Given that AST it constructs a GraphQLSchema. The resulting schema
- * has no resolve methods, so execution will use default resolvers.
+ * Given that AST it return object that can be used to construct GraphQLSchema.
  */
-export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
+export function buildASTSchemaConfig(ast: DocumentNode) {
```

Comment:
@wincent Thank you. I used "GraphQLSchemaComponents" as the name for the new type and also rename function to `buildASTSchemaComponents`.